napari/utils/theme.py

```diff
@@ -156,11 +181,27 @@ def get_theme(name):
     """
     if name in _themes:
         theme = _themes[name]
-        return theme.copy()
+        _theme = theme.copy()
+        if as_dict:
+            warnings.warn(
```

Comment:
quick note here, with this warning, everyone will be seeing a warning on every napari usage now because of napari-console:
```
/Users/talley/miniconda3/envs/napdev/lib/python3.9/site-packages/napari_console/qt_console.py:161: FutureWarning: Themes were changed to use evented model with Pydantic's color type rather than the `rgb(x, y, z)`. You can get the old color by calling `color.as_rgb()`. The `as_dict=True` option will be removed in 0.X.X
  theme = get_theme(self.viewer.theme)
```

Reply:
That's true - its can be very easily fixed in the `napari-console` plugin but until then it will be showing each time. 

The startup warning is actually caused by the fact that I am connecting the `theme.console` and `theme.syntax_style` events directly to the `qt_viewer.console._update_theme` methods. I could wrap with `warnings.catch_warnings` for the time being?

Reply:
I do think it's pretty jarring to have everyone see a warning in our default install.  So either here, or in a follow-up PR before we release, we should either do the catch_warnings... or we should update napari-console and pin our dependency here